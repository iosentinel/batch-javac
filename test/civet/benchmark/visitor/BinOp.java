package benchmark.visitor;

public class BinOp {
	int apply(int x, int y) {
		return this.apply(x, y);
	}
}
