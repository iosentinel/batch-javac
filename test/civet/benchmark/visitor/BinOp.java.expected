package benchmark.visitor;

public class BinOp {
    
    public BinOp() {
        super();
    }
    
    int apply(int x, int y) {
        return this.apply(x, y);
    }
}